### PR TITLE
feat(cloud.project.training): add training region GPU endpoint

### PR DESCRIPTION
**Pull request by co42**

### CODE REVIEW

File: src/api/cloud/project/ai/capabilities/training/region/cloud-project-ai-capabilities-training-region.service.js [hunk 1]
```diff
@@ -2,4 +2,5 @@ angular
   .module('ovh-api-services')
   .service('OvhApiCloudProjectAiCapabilitiesTrainingRegion', ($injector) => ({
     v6: () => $injector.get('OvhApiCloudProjectAiCapabilitiesTrainingRegionV6'),
+    gpu: () => $injector.get('OvhApiCloudProjectAiCapabilitiesTrainingRegionGpu'),
```
Comment (antleblanc) [inline src/api/cloud/project/ai/capabilities/training/region/cloud-project-ai-capabilities-training-region.service.js:5]:
```suggestion
    Gpu: () => $injector.get('OvhApiCloudProjectAiCapabilitiesTrainingRegionGpu'),
```

**Review by antleblanc — DISMISSED** [review]
Thank you @co42!

Changes looks good to me. Just leave a suggestion about title case.

**Edit:** Suggestion has been applied.

**Review by antleblanc — APPROVED** [review]

**Comment by antleblanc** [comment]
@co42 Could you please let us know when endpoint api will be available?
Thank you!

**Review by varun257 — APPROVED** [review]

**Comment by co42** [comment]
@antleblanc The enpoint API should already be available

File: src/api/cloud/project/ai/capabilities/training/region/gpu/cloud-project-ai-capabilities-training-region-gpu.v6.service.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+angular
+  .module('ovh-api-services')
+  .service('OvhApiCloudProjectAiCapabilitiesTrainingRegionGpuV6', ($resource) => {
+    const resource = $resource('/cloud/project/:serviceName/ai/capabilities/training/region/:region/gpu', {
+      serviceName: '@serviceName',
+      region: '@region',
+    });
+
+    return resource;
+  });
```
Comment (JeremyDec) [inline src/api/cloud/project/ai/capabilities/training/region/gpu/cloud-project-ai-capabilities-training-region-gpu.v6.service.js:10]: **nitpick**
```suggestion
  .service('OvhApiCloudProjectAiCapabilitiesTrainingRegionGpuV6', ($resource) => $resource('/cloud/project/:serviceName/ai/capabilities/training/region/:region/gpu', {
      serviceName: '@serviceName',
      region: '@region',
    }));
```

**Review by JeremyDec — APPROVED** [review]

**Review by JeremyDec — APPROVED** [review]

**Pull request merged**